packages/experimental/Button/src/Button.types.ts

```diff
@@ -79,6 +56,11 @@ export interface ButtonTokens extends FontTokens, IBorderTokens, IShadowTokens {
   subtle?: ButtonTokens;
   fluid?: ButtonTokens;
   fab?: ButtonTokens;
+  small?: ButtonTokens;
+  medium?: ButtonTokens;
+  large?: ButtonTokens;
+  hasContent?: ButtonTokens;
+  hasIcon?: ButtonTokens;
```

Comment:
These tokens would affect how the button is styled if these states were active (which is controlled by that lookup function in button.tsx).

i.e. right now by default hasContent by itself doesn't do anything by default, but if you wanted to say change the backgroundColor of the button when the button has content, this token is how you would change that.

Mostly it's here to adjust styling of certain things when the button has content. We can do it in the Button render function instead, but I thought that this way would be better since then the styling is handled mostly in the same space.